test/hotspot/jtreg/vmTestbase/nsk/share/jni/jni_tools.h

```diff
@@ -133,8 +131,9 @@ int nsk_jni_check_exception(JNIEnv* jni, const char file[], int line);
  * If value is negative, the first character of the stored string is
  * the minus sign (-). The function returns a pointer to the begining
  * of the result string.
+ * Declared and defined in jvmti_common.h
  */
-char *jlong_to_string(jlong value, char *string);
+//char *jlong_to_string(jlong value, char *string);
```

Comment:
It seems odd to have jlong_to_string and julong_to_string implemented in different places. Maybe more cleanup is needed here.

Reply:
I'd suggest to move julong_to_string to the same place where the jlong_to_string is placed and remove all these with the comments from jni_tools.

Reply:
I moved julong_to_string (and define for julong type) to jvmti_common.h
About JVMTITools.*
JVMTITools.cpp is included in 628 tests, JVMTITools.h is included in 348 files.
I don't think removing this 2 files worth such massive change